flux/flux.d.ts

```diff
@@ -71,21 +73,43 @@ declare module "flux" {
 declare namespace FluxUtils {
 
     import React = __React;
+    import fbEmitter = FBEmitter;
+    import immutable = Immutable;
+    
+    /**
+     * Default options to create a Container with
+     * 
+     * @interface RealOptions
+     */
+    interface RealOptions {
```

Comment:
All properties are optional. So, you should at each optional property add a `?` at the end of the property name.